data/_ui/tooltips.txt

```diff
@@ -667,10 +667,10 @@ tip "shield protection:"
 	`Protection against shield damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in shield damage, while a total value of 11 only results in a 10 percent reduction.`
 
 tip "silent jumps:"
-	`Prevents a ship from making hyperdrive or jump drive sounds when entering or leaving a system.`
+	`Prevents a ship from playing hyperdrive/jump sounds even the default sounds that are played when no other sound is defined.`
```

Comment:
Huh? This line completely confuses me. Is the attribute supposed to disable the default hyperdrive/jump drive sounds? It kind of reads like it's a band-aid fix for a bug where hyperdrive sounds play in the background non-stop.

Reply:
That's what warp was wondering about in the old PR.